src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -103,6 +105,8 @@ let default : conf =
     output_format = Output_format.Text;
     logging_level = Some Logs.Warning;
     force_color = false;
+    max_chars_per_line = Constants.default_max_chars_per_line;
```

Comment:
I usually get rid of the Constants.default_xxx constants and instead inline the value there,
when the constant is never used outside the CLI configuration (e.g., see max_target_bytes).

Reply:
done in 74e80af3d